### ISSUE TITLE
Set sort order with group by?

### ISSUE THREAD

**Issue by lapidus** [angular-ui/ui-select#356]
I want to group countries by "type" but it doesn't sort these option groups alphabetically.
How can I set a sort order of the groups created?

```
<ui-select-choices group-by="'type'" repeat="country in countryList | propsFilter: {name: $select.search}">
```

Thanks!

**Comment by burzum** [comment]
+1

**Comment by netstyler** [comment]
+1

**Comment by Vitaliy-Yarovuy** [comment]
+1

**Comment by Vitaliy-Yarovuy** [comment]
i have create pull request for that
https://github.com/angular-ui/ui-select/pull/840

**Comment by brianfeister** [comment]
This was merged at #969
